src/codewhisperer/service/keyStrokeHandler.ts

```diff
@@ -75,6 +70,9 @@ export class KeyStrokeHandler {
 
             let triggerType: CodewhispererAutomatedTriggerType | undefined
             const changedSource = new DefaultDocumentChangedType(event.contentChanges).checkChangeSource()
+            // Time duration between 2 invocations should be greater than the threshold
```

Comment:
Could we write a simple test for `processKeyStroke` to catch regressions?

Reply:
No problem. I am working on it